published/012-2018-Oct-2.md

```diff
@@ -0,0 +1,63 @@
+
+## IPFS in the wild
+*Do you follow [IPFS on Twitter](https://twitter.com/IPFSbot)? For the latest mentions of IPFS in the news, check our Twitter feed or see the [latest articles on Awesome IPFS](https://awesome.ipfs.io/categories/articles/).* 
+
++ [Build a Decentralized Chat App with Knockout and IPFS](https://medium.com/textileio/build-a-decentralized-chat-app-with-knockout-and-ipfs-fccf11e8ce7b), Textile.io, 28 Sep 2018
++ [IPFS: A Complete Analysis of The Distributed Web](https://medium.com/zkcapital/ipfs-the-distributed-web-e21a5496d32d), zK Capital, 28 Sep 2018 *(Clicking this link means you are already using IPFS!)*
++ [Introduction to IPFS: The New & Improved Web](https://medium.com/@rtradetechnologies/introduction-to-ipfs-the-new-improved-web-efc89f158e6f), 26 Sep 2018
++ [Crypto-Advocacy Comes Of Age As Trade Groups And Think Tanks Move On Washington](https://www.forbes.com/sites/astanley/2018/09/22/crypto-advocacy-comes-of-age-as-trade-groups-and-think-tanks-move-on-washington/#67ac823666b5), Forbes Magazine, 22 Sep 2018
++ [Presenting a United Front of Blockchain Companies to Work With Congress](https://bitcoinmagazine.com/articles/presenting-united-front-blockchain-companies-work-congress/), Bitcoin Magazine, 21 Sep 2018
++ [Walking the DWeb walk](https://medium.com/textileio/walking-the-dweb-walk-9abc8b6e2b20) – How we moved our website to secure IPFS thanks to Cloudflare, Textile.io, 20 Sep 2018
++ [What’s really happening when you add a file to IPFS?](https://medium.com/textileio/whats-really-happening-when-you-add-a-file-to-ipfs-ae3b8b5e4b0f), 28 Aug 2018
++ [IPFS & Libp2p Developer Meetings - Growing The Decentralized Tech That Helps OpenBazaar Work](https://openbazaar.org/blog/IPFS-Libp2p-Developer-Meetings-Growing-The-Decentralized-Tech-That-Helps-OpenBazaar-Work/), OpenBazaar, 30 Jul 2018
+
+## Updates and new releases
+*See the latest releases of IPFS tools and projects across the ecosystem.*
+
++ [ipld-explorer-cli 0.14](https://blog.ipfs.io/44-ipld-explorer-cli-0-14/) - IPLD (Inter Planetary Linked Data) is the underlying data structure used by IPFS that allows it to store, retrieve and traverse through any type of content-addressed data.
++ [ipfs-cluster 0.6.0](https://github.com/ipfs/ipfs-cluster/blob/0.6.0/changelog/CHANGELOG.md) – Increased the minor release number to signal changes to the Go APIs after upgrading to the new cid package, but, other than that, this release does not include any major changes.
++ [js-ipfs 0.32.3](https://github.com/ipfs/js-ipfs/releases/tag/v0.32.3) – Some bug fixes for the JavaScript implementation of the IPFS protocol.
++ [qri-io 0.4.0](https://github.com/qri-io/frontend/releases/tag/v0.4.0) - qri electron & web frontend, react & redux webapp for qri.io, plus [check out the screencast demo](https://www.youtube.com/watch?v=3CSPA3qGU5g)!
```

Comment:
qri is already at 0.5.4, the 0.4.0 release was in june :)

Reply:
Ahh, gotcha. I’ve been seeing a lot about 0.4.0 for some reason in my timeline. Thanks!